generators/app/templates/src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+interface Props {
+  checked: boolean;
+  className?: string;
+  disabled?: boolean;
+  error?: string;
+  errorClassName?: string;
+  inputClassName?: string;
+  label?: string;
+  labelClassName?: string;
```

Comment:
This props are optional but have a default value.
If they are actually optional, i'd remove de default value, otherwise remove the optional marker.

Reply:
Yes, this make sense with propTypes. But, in this case I see it well for the prolificity of the component.
If i make mandatory the optional props, the developer have to pass some value obligatorily.

Reply:
👍 it made me think that there was some incongruence defining a prop as optional (so letting anybody omit that prop on purpose) but assigning a default value. I discussed it with @pabloferro and explained me some use cases for that.
So we are ok.